src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java

```diff
@@ -236,4 +240,77 @@ private static boolean checkIsAllLinesAreWhitespace(String[] lines, int lineFrom
         }
         return result;
     }
+
+    /**
+     *  Checks for empty LITERAL_CASE.
+     *
+     *  @param ast a {@code DetailAST} value
+     */
+    protected void emptyCaseCheck(DetailAST ast) {
+        final DetailAST slistToken = ast.getNextSibling();
+        if (slistToken != null
+                && slistToken.getType() == TokenTypes.SLIST) {
+            final boolean emptyCase;
+            if (option == BlockOption.STMT) {
+                emptyCase = !caseHasStmt(slistToken);
+                if (emptyCase) {
+                    log(ast.getLineNo(),
+                            ast.getColumnNo(),
+                            MSG_KEY_BLOCK_NO_STMT,
+                            ast.getText());
+                }
+            }
+            else {
+                emptyCase = !caseHasText(slistToken);
+                if (emptyCase) {
+                    log(ast.getLineNo(),
+                            ast.getColumnNo(),
+                            MSG_KEY_BLOCK_EMPTY,
+                            ast.getText());
+                }
+            }
+        }
+    }
+
+    /**
+     * Helps in checking if case has any statements.
```

Comment:
Rephrase. This method is not helper. It is a part of logic. It does not help in checking, it checks sth.